aspnetcore/blazor/get-started.md

```diff
@@ -16,12 +16,15 @@ By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.
 
 Get started with Blazor:
 
-1. Install the latest [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0) release.
+1. Install tooling:
 
-1. Install the [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) template by running the following command in a command shell:
+   * For an experience in Blazor Server, download and install the latest [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0) release.
+   * For an experience in Blazor WebAssembly, download and install the [.NET Core 3.1 Preview SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1) and [Visual Studio 16.4 Preview 2 or later](https://visualstudio.microsoft.com/vs/preview/).
+
+1. Install the [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) template by running the following command in a command shell. The [Microsoft.AspNetCore.Blazor.Templates](https://www.nuget.org/packages/Microsoft.AspNetCore.Blazor.Templates/) package has a preview version while Blazor WebAssembly is in preview.
```

Comment:
In the 3.0 doc set, installing the Blazor WebAssembly has two sub steps: install .NET Core 3.1 and then run the command to install the template.

Reply:
We also need to update the Visual Studio specific steps in the 3.0 doc set to clarify that if you're using Blazor WebAssembly, then you need to install the latest VS 2019 16.4 preview. in the 3.1 doc set we need to tell folks using both Blazor Server and Blazor WebAssembly to VS 2019 16.4 preview.